myosuite/envs/myo/myochallenge/__init__.py

```diff
@@ -33,28 +33,53 @@ def register_env_with_variants(id, entry_point, max_episode_steps, kwargs):
 
 register_env_with_variants(id='myoChallengeBimanual-v0',
         entry_point='myosuite.envs.myo.myochallenge.bimanual_v0:BimanualEnvV1',
-        max_episode_steps=250,
+        max_episode_steps=300,
         kwargs={
             'model_path': curr_dir + '/../assets/arm/myoarm_bionic_bimanual.xml',
             'normalize_act': True,
             'frame_skip': 5,
+            'obj_scale_change': [0.1, 0.05, 0.1],  # 10%, 5%, 10% scale variations in respective geom directions
```

Comment:
respect to what?
> # 10%, 5%, 10% scale variations in XYZ geom directions respectively

Reply:
@Balint-H can you help take a look here?

Reply:
Respect to starting values

Reply:
also mention the order X, Y, Z